### PR TITLE
Luke/add task cleanup script

### PR DESCRIPTION
**Pull request by lukebrogan-mend**
Add script to remove legacy pending tasks in Mend SCA UI.

### CODE REVIEW

**Comment by tabacmend** [comment]
bash pending-task-cleanup.sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   325    0    87  100   238    203    557 --:--:-- --:--:-- --:--:--   770
Warning: No uuids found in the API response.

Got some pending tasks in my org

**Review by tabacmend — DISMISSED** [review]

**Comment by lukebrogan-mend** [comment]
@samq-ws  - can we please get this merged to main? tested all ok.

File: Scripts/Mend SCA/README.md [hunk 1]
```diff
@@ -273,7 +273,37 @@ The [cache-ua.sh](cache-ua.sh) script can be added to the CI/CD pipeline on a st
...
+This script allows the user to cleanup outstanding [Pending Tasks](https://docs.mend.io/bundle/wsk/page/ui_-_request_history_report_and_pending_tasks.html) in the Mend SCA UI, when this process is no longer required. Please ensure the setting '[Open pending tasks for new libraries](https://docs.mend.io/bundle/wsk/page/ui_-_request_history_report_and_pending_tasks.html)' is disabled under the Integrate, Advanced Settings area. In addition, also ensure that their are no [Policies](https://docs.mend.io/bundle/sca_user_guide/page/managing_automated_policies.html#Applying-Actions-to-a-Library) that are 'Reassign' or 'Condition', which could create new tasks. 
+
+The [pending-task-cleanup.sh](pending-task-cleanup.sh) script is designed to be executed one time per organisation to clean up historic pending requests the Mend SCA UI. 
+
+<br>
+
+**Prerequisites:**  
+
+* `jq` and `curl` must be installed
+* The tasks within Mend should be assigned to a user and not to a group (Edit policy->Reasssign->Assign to User)
```
Comment (samq-ws) [inline Scripts/Mend SCA/README.md:297]: What is the reason why for this?  It is not typical for a task to always be assigned to a user.

File: Scripts/Mend SCA/README.md [hunk 2]
```diff
@@ -273,7 +273,37 @@ The [cache-ua.sh](cache-ua.sh) script can be added to the CI/CD pipeline on a st
...
+
+<br>
+
+**Execution:**  
+
+```
+export MEND_URL=example-https://saas.mend.io
```
Comment (samq-ws) [inline Scripts/Mend SCA/README.md:304]: remove example- and just leave the example

**Review by samq-ws — CHANGES_REQUESTED** [review]

File: Scripts/Mend SCA/README.md (hunk 1 above)
Comment (lukebrogan-mend) [reply Scripts/Mend SCA/README.md:297]: the API returns 0 if the tasks are not assigned to a user.

File: Scripts/Mend SCA/README.md (hunk 2 above)
Comment (lukebrogan-mend) [reply Scripts/Mend SCA/README.md]: done

**Review by samq-ws — APPROVED** [review]

**Pull request merged**